web/locales/de.json

```diff
@@ -17,6 +40,11 @@
         "addeditsource": "<a href=\"%s\" target=\"_blank\">Quelle bearbeiten oder hinzufügen</a>",
         "helpfrom": "mit Unterstützung von"
     },
+    "left-panel": {
+      "zone-list-header-title": "Klimaauswirkung nach Gegend",
+      "zone-list-header-subtitle": "Sortiert nach Kohlenstoffintensität der verbrauchten Elektrizität (gCO2äq/kWh)",
+      "search": "Gegenden suchen"
```

Comment:
The current German is:

        "zone-list-header-title": "Klimaeinfluss der Regionen",
        "zone-list-header-subtitle": "Sortiert nach CO2-Intensität der verfügbaren Elektrizität (gCO2äq/kWh)",
        "search": "Land oder Region suchen"